channels/chan_simpleusb.c

```diff
@@ -347,6 +348,7 @@ static struct chan_simpleusb_pvt simpleusb_default = {
 	.rxondelay = 0,
 	.txoffdelay = 0,
 	.pager = PAGER_NONE,
+	.serial = "",
```

Comment:
I'm not sure about this since the variable is "char serial[14]".  Also, with the default struct being zero'd out then is this needed (kinda the same way that .devstr is not being initialized)?